src/ray/gcs/gcs_client/test/service_based_gcs_client_test.cc

```diff
@@ -797,28 +812,66 @@ TEST_F(ServiceBasedGcsClientTest, TestErrorInfo) {
   ASSERT_TRUE(ReportJobError(error_table_data));
 }
 
-TEST_F(ServiceBasedGcsClientTest, TestDetectGcsAvailability) {
-  // Create job table data.
-  JobID add_job_id = JobID::FromInt(1);
-  auto job_table_data = Mocker::GenJobTableData(add_job_id);
+TEST_F(ServiceBasedGcsClientTest, TestJobTableReSubscribe) {
```

Comment:
Add a comment: `Test that subscription of the job table can still work when GCS server restarts.`

Reply:
fixed